src/tools/clippy/clippy_dev/Cargo.toml

```diff
@@ -4,11 +4,11 @@ version = "0.0.1"
 edition = "2021"
 
 [dependencies]
-aho-corasick = "0.7"
+aho-corasick = "1.0"
```

Comment:
I think clippy would also prefer these changes being made in the clippy repository directly. Could you revert them too?